examples/tailwind-css/src/index.tsx

```diff
@@ -9,11 +9,28 @@ import reportWebVitals from './reportWebVitals';
 const rootElement = document.getElementById('root');
 const root = createRoot(rootElement!);
 
+const theme = createTheme({
```

Comment:
We may as well customize the theme a bit here, so that prople who use custom theme will be able to easily connect the dots. Cc @danilo-leal for opinion.

Reply:
Can you elaborate a bit more?! Not sure if I'm getting the context correctly!

Reply:
Here we are creating a theme only for setting the contain for the popover components. I was thinking that we can also add some palette values so that people know that the API is the same. I can see that I just created confusion with this, so I am resolving the comment.